batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/exception/ValidationException.java

```diff
@@ -0,0 +1,24 @@
+package com.lftechnology.batch7crud.exception;
+
+import java.util.Map;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/26/16.
+ */
+public class ValidationException extends Exception {
+  private Map<String, String> errors; //NOSONAR
+
+  public ValidationException(){
+    super("Validation Error");
+  }
+
+  public ValidationException(String message){
+    super(message);
+  }
+
+  public ValidationException(Map<String, String> errors) { this.errors = errors; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)